pkg/master/master.go

```diff
@@ -569,6 +568,13 @@ func (m *Master) init(c *Config) {
 	nodeStorage, nodeStatusStorage := nodeetcd.NewREST(dbClient("nodes"), storageDecorator, c.KubeletClient, m.proxyTransport)
 	m.nodeRegistry = node.NewRegistry(nodeStorage)
 
+	podStorage := podetcd.NewStorage(
+		dbClient("pods"),
+		storageDecorator,
+		nodeStorage,
```

Comment:
I'd rather have a different object than have `nodeStorage` satisfy this interface, but at least do `client.ConnectionInfoGetter(nodeStorage)` to clarify why we're giving `nodeStorage` to `podStorage`